### PR TITLE
Basic one page homesite

### PR DESCRIPTION
**Pull request by NazarKY**

### CODE REVIEW

**Comment by NazarKY** [comment]
Have [deployed](http://staging.nazarky-oauth2me.divshot.io/) on divshot service.

@tehhi42 pls [verify](http://staging.nazarky-oauth2me.divshot.io/)

**Comment by tehhi42** [comment]
My comments:
1. I think it would be great to add the link to our website and/or to Dima's github. Or Jetthoughts github. 
2. 

3. 

4.

**Comment by tehhi42** [comment]
Current to-dos:
- [x] correct OAuth2.me everythere on the page
- [x] choose a slogan
- [ ] find an icon for the header
- [ ] do screenshots of setup
- [ ] find a picture for the 1st block (infographics?)
- [ ] add credentials in the footer

**Comment by tehhi42** [comment]
Slogan variants:
- Be free from domain name.
- Free from domain name. 
- Domain name unchained.
- Remove oauth borders.
- Break free of domain name.
- Break free of name
- break name's jail 
- Break oauth setup bureaucracy 
- Domain Name Independence 
- Independence from domain name 
- Your OAuth Multipass

**Comment by NazarKY** [comment]
@pftg  pls review.

File: index.html [hunk 1]
```diff
@@ -1,10 +1,144 @@
 <html>
 <head>
-  <title>OAuth2 me</title>
-  <link href="//fonts.googleapis.com/css?family=Noto+Sans:400italic,400,700italic,700" rel="stylesheet" type="text/css">
-  <link href="/assets/stylesheets/main.css" rel="stylesheet" type="text/css">
+  <title>Oauth2me</title>
```
Comment (pftg) [inline index.html]: `OAuth2.me - Domain Name Independence`

File: index.html [hunk 2]
```diff
@@ -1,10 +1,144 @@
...
+    <!-- Fixed navbar -->
+    <div class="navbar navbar-default navbar-fixed-top">
+      <div class="container">
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <a class="navbar-brand" href="#"><i class="fa">Oauth2.me</i></a>
```
Comment (pftg) [inline index.html]: OAuth2.me

**Comment by pftg** [comment]
а где можно найти онлайн эту старницу

**Comment by pftg** [comment]
@NazarKY please place link before code review

**Comment by NazarKY** [comment]
@pftg pls [verify](http://staging.nazarky-oauth2me.divshot.io/)

**Comment by NazarKY** [comment]
@pftg [verify](http://staging.nazarky-oauth2me.divshot.io/)

**Comment by NazarKY** [comment]
@pftg [verify](http://staging.nazarky-oauth2me.divshot.io/)

**Comment by tehhi42** [comment]
@pftg @NazarKY My suggestions:

**Comment by tehhi42** [comment]
A question:

**Comment by tehhi42** [comment]
@pftg @NazarKY I have made 2 variants of a picture instead of mobile phone icon for the block "What is it?" 
untitled design
untitled design 1

**Comment by tehhi42** [comment]
untitled design 2

**Comment by tehhi42** [comment]
untitled design 3

**Comment by NazarKY** [comment]
The latest version is [here](http://staging.oauth2-me.divshot.io/)

**Comment by tehhi42** [comment]
@NazarKY

**Comment by tehhi42** [comment]
@NazarKY Created mockup based on Paul's comments. https://jetthoughts.mybalsamiq.com/projects/oauth2me/grid

**Comment by tehhi42** [comment]
The setup process should be represented with the help of pictures. For reference: 
 
Ruby code form keeper should look like this:

**Comment by Magons** [comment]
@NazarKY and @tehhi42 pls [review](http://development.ono2.divshot.io/)
I can still experiment with images, just say, they should be more bigger or smaller, more clear or more vague?

**Comment by tehhi42** [comment]
IMHO, slightly bigger. 
 Enough vagueness for me :-)

**Comment by Magons** [comment]
@tehhi42 pls [review](http://development.ogogo.divshot.io/)

**Comment by tehhi42** [comment]
Looks good to me. @pftg your opinion?

**Comment by pftg** [comment]
@tehhi42 as for me better to have much bigger, I do not see, what's on the pictures even on 24" screen.
And font should have the same size like for others texts

**Comment by pftg** [comment]
We have too small text amount at the right, so maybe we should place it under/above pictures to be as captions, alos about sizes and layout: http://i.imgur.com/djxsAHI.jpg

**Comment by Magons** [comment]
@pftg and @tehhi42 pls [review] (http://development.ogo.divshot.io/)

**Comment by Magons** [comment]
@pftg and @tehhi42 pls [review](http://development.ogo.divshot.io/)

**Comment by pftg** [comment]
@tehhi42 why we chose such images?

**Comment by tehhi42** [comment]
@pftg @Magons Can we use actual screenshot of the installation process (for YOUR OWN OAUTH2ME ON GITHUB)?

**Comment by Magons** [comment]
@tehhi42 I think not, because there is little code.

**Comment by pftg** [comment]
@tehhi42 any updates?

**Comment by tehhi42** [comment]
@Magons it is ok that there will be a little code. Our aim is to show the actual installation process.

**Comment by Magons** [comment]
@tehhi42 For this `For each new projects you should go to on OAuth2 provider site and setup new app. So with this gem you do not need it at all.` text I suggest use screenshot of adding app on OAuth2 provider site. For other text use screenshot from site rubygems.org of our gem.
All code of install we display in section before and there are only one line of code.
@pftg your opinion?

**Comment by pftg** [comment]
on first screen you should take screenshot of gemfile, on the second you should add screenshot with Heroku button

**Comment by Magons** [comment]
@pftg @tehhi42 pls [review](http://development.newimages.divshot.io/)

**Comment by tehhi42** [comment]
@Magons ok, let's deploy it.

**Comment by tehhi42** [comment]
@Magons Please change the content according to @tmwh comments. 
"В platform-independent setup нужно поместить вот это:

To use oauth2.me service you need to:
1. Set your redirect uri to http://oauth2.me/redirect
2. Your application should use the state parameter to pass your redirect URI encoded with base64

В YOUR OWN OAUTH2ME ON GITHUB вот это:

How to install your own OAuth2.me?
Make a fork of our repository https://github.com/tmwh/oauth2-me-ruby/fork
Remove the `CNAME` file from the root.
Wait until github will compile the website for you.

А то, что сейчас в YOUR OWN OAUTH2ME ON GITHUB - это отдельный раздел выходит, типа как его можно использовать."

**Comment by Magons** [comment]
@tehhi42 pls [review](http://development.newtabownoauth.divshot.io)

**Comment by tehhi42** [comment]
@Magons verified.

**Pull request merged**